packages/chat-stateful-client/src/StatefulChatThreadClient.ts

```diff
@@ -109,6 +109,14 @@ class ProxyChatThreadClient implements ProxyHandler<ChatThreadClient> {
       case 'listReadReceipts': {
         return createDecoratedListReadReceipts(chatThreadClient, this._context);
       }
+      case 'sendTypingNotification': {
```

Comment:
there is a unit test for statefulChatThreadClient:
StatefulCallClient.test.ts in the same folder

And there is a test set named 'errors should be reported correctly from Call when'

Would be nice to add this into the test set